apis/appmesh/v1beta2/virtualgateway_types.go

```diff
@@ -92,6 +92,22 @@ type VirtualGatewayHealthCheckPolicy struct {
 	UnhealthyThreshold int64 `json:"unhealthyThreshold"`
 }
 
+// VirtualGatewayConnectionPool refers to the connection pools settings for Virtual Gateway.
+// Connection pool limits the number of connections that an Envoy can concurrently establish with
+// all the hosts in the upstream cluster. Currently connection pool is supported only at the listener
+// level and it is intended protect your local application from being overwhelmed with connections.
+type VirtualGatewayConnectionPool struct {
+	// Specifies http connection pool settings for the virtual node listener
```

Comment:
virtual node -> virtual gateway(same for http2 & grpc below). No limit on inbound tcp connections?

Reply:
oops, thanks for catching this. Let me fix

Reply:
On why we don't have TCP connection pools for virtual gateway, we currently do not support TCP listeners on virtual gateways: https://github.com/aws/aws-app-mesh-roadmap/issues/233